client/scripts/com/monsters/maproom_advanced/MapRoomCell.as

```diff
@@ -326,6 +327,7 @@ package com.monsters.maproom_advanced
          this._updated = true;
          this._processed = false;
          this._base = serverData.b;
+         this._cell_id = serverData.cell_id;
```

Comment:
Reminder: Remove any new client side logic, we should emulate the original server.